src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  
+
```

Comment:
I do not know why this empty line is here however my bad for not spotting it before pushing

Reply:
It doesn't matter too much. _Not_ having whitespace is more of a problem as that is more likely to affect readability.